gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -50,12 +50,16 @@ message GetNodeInfoResponse {
 }
 
 message PayInvoiceRequest {
+  // Bolt11 invoice
   string invoice = 1;
 
+  // The maximum delay, in blocks, that the payment route can be locked for
   uint64 max_delay = 2;
 
+  // The maximum fee, in millisats, that will be paid as a lightning routing fee
   uint64 max_fee_msat = 3;
 
+  // The hash of the payment
```

Comment:
Can you add that this is needed for `pay_private`?

Reply:
Done. Leaving unresolved since I don't have a good answer to the question above.